src/gsAssembler/gsOverIntegrateRule_.cpp

```diff
@@ -0,0 +1,10 @@
+#include <gsCore/gsTemplateTools.h>
```

Comment:
There is not hpp file.
Either create an hpp file and insert some code in there, or remove the gsOverIntegrateRule_.cpp  since it is not needed without the hpp file

Reply:
Removed the _cpp